Examples/draggable-graph.lisp

```diff
@@ -114,10 +115,8 @@
 (define-presentation-to-command-translator record-dragging-translator
     (t com-drag-node draggable-graph-demo
        :tester ((object presentation)
-                (declare (ignore object))
                 (find-graph-node presentation)))
```

Comment:
So the convoluted macro-hackery behing define-presentation-to-command-translator creates a bunch of nested forms before including the body. The upshot is that we end up trying to ignore a variable that was declared in an outer let block. I fixed this by changing the macro such that its kw args to tester are ignorable. And, in so doing, I've removed these declarations that were, justifiably, triggering warnings.